share/spice/snow/snow.js

```diff
@@ -53,202 +53,104 @@ function nrio (api_result) {
     
     var $dom = Spice.getDOM('snow');
     $dom.css({
-        background: '#0C0F2B',
+        background: '#D0D0D0',
```

Comment:
Let's stick with the original navy blue. It was much easier to read and see the snow against.